src/components/Input.tsx

```diff
@@ -1,16 +1,24 @@
 import React from "react";
 
 const CreateInput: React.FC<{
-  type?: string;
-  value: any;
-  changeState?: Function;
+  type: string;
+  value: string;
+  changeState: React.Dispatch<React.SetStateAction<string>>;
   className?: string;
-}> = ({ type, value, className }) => {
+}> = ({ type, value, changeState, className }) => {
```

Comment:
remove the className prop. Instead of creating any type of input createText input if it makes you feel dry ☂️ otherwise make it just... jsx/tsx 🤷

Reply:
I wanted this className as optional prop, because two types of input use this component, so we can add some conditional class. Maybe I don't understand, but should I make separate component for every type of input?